src/main/java/tools/image/TileImageReconstructor.java

```diff
@@ -27,6 +27,7 @@
 import games.strategy.triplea.ui.screen.TileManager;
 import games.strategy.ui.Util;
 import games.strategy.util.PointFileReaderWriter;
+import javax.swing.filechooser.FileFilter;
```

Comment:
I think this is the Checkstyle violation causing the build break: "Import statement for 'javax.swing.filechooser.FileFilter' is in the wrong order. Should be in the 'STANDARD_JAVA_PACKAGE' group, expecting not assigned imports on this line."